### PR TITLE
Feature: Update comments by Posting API

### PR DESCRIPTION
**Pull request by luftedar**
# Feature: Post Comments to the API
In this update:
- Added 'post' for the comments.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -46,26 +46,33 @@ const displayMovie = async() => {
...
 }
 
 const getComments = async (movieId) => {
   const comments = await axios.get(`https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/XMHWey4za3iNnBFD5KUq/comments?item_id=${movieId}`)
   return comments.data;
 }
 
-const showComments = () => {
+const showComments = async (id) => {
+  console.log(id);
```
Comment (Wes-Isaac) [inline src/index.js]: - [x] Please remove console logs from your code they might contain private data and makes the site slow.

File: src/index.js [hunk 2]
```diff
@@ -46,26 +46,33 @@ const displayMovie = async() => {
...
+  try{ getCommentsFromAPI = await getComments(id)}catch{
+    popUpHtml(id);
+    commentHTML = '<p>No Comments Yet</p>'
+    getCommentsFromAPI = [];
+  };
```
Comment (Wes-Isaac) [inline src/index.js]: - [x] This try and catch block doesn't take action when the API responds with a 400. Please fix that.

File: src/index.js [hunk 3]
```diff
@@ -76,13 +83,38 @@ const showComments = () => {
...
+  }
+  else if (e.target.classList.contains('fa-heart')) {
+    updateLikes(e.target)
+  }else if (e.target.type === 'button'){
+    let sentID = (e.target.id).toString();
+    let sentUserName = (e.target.parentNode.childNodes[1].value);
+    let sentUserComment = (e.target.parentNode.childNodes[3].value);
     comments.innerHTML = `
     <div id='pop'>
       <img src="${movies[parseInt(e.target.parentNode.id) - 1].image.medium}" alt="${movies[parseInt(e.target.parentNode.id) - 1].name}">
```
Comment (Wes-Isaac) [inline src/index.js]: - [x] The index you're passing to the movies array is not a number, please fix that.

File: src/index.js [hunk 4]
```diff
@@ -76,13 +83,38 @@ const showComments = () => {
         <form action="submit" id="form-area">
           <input type="text" id="name" placeholder="Your Name">
           <textarea type="textarea" rows="4" cols="50" name="comment">Your Insights</textarea>
-          <button id="form-submit" type="button">Submit Comment</button>
+          <button id="${id}" type="button">Submit Comment</button>
```
Comment (Wes-Isaac) [inline src/index.js]: - [x] All id attributes should be unique in the code. Please make all id's unique.

**Review by Wes-Isaac — CHANGES_REQUESTED** [review]
# Status: Changes Required ♻️ 

Hello @luftedar, I am Isaac, your reviewer for this PR

Well done so far with your project!

Your project is almost perfect, congratulations! There are some really small details to correct before moving on, good luck!

I will mention to you a small summary of things that need to be changed in order to keep moving:

 Please check the comments below in this PR.

**Comment by luftedar** [comment]
### Hi @Wes-Isaac 
Fixed the bugs and implement required changes :+1:

**Comment by Wes-Isaac** [comment]
# Status: Approved 💯 
You've done a great job of implementing project requirements.

It is time to merge 🥇

**Review by Wes-Isaac — APPROVED** [review]

**Pull request merged**